packages/cli/internal/pkg/cli/workflow/workflow_tasks_test.go

```diff
@@ -200,6 +201,58 @@ func (s *GetWorkflowTasksTestSuite) TestGetWorkflowTasks_WithTaskNameFailure() {
 	}
 }
 
+func (s *GetWorkflowTasksTestSuite) TestGetWorkflowTasks_WithExitCodeUnknown() {
```

Comment:
NIT: Please update this to "TestGetWorkflowTasks_WithExitCodeNil"

Reply:
no prob.